Automattic/wp-calypso
# Remove concierge item from Jetpack plans

#### Changes proposed in this Pull Request

* Remove concierge item from Jetpack plans, it no longer applies

#### Testing instructions



* Connect a new site and get to plans page
* Observe concierge is no longer offered

This screenshot shows location of concierge item:

<img width="921" alt="concierge plan feature" src="https://user-images.githubusercontent.com/51896/63466200-a3a06300-c417-11e9-9ae0-23db7c3604b9.png">
